.github/CHANGELOG.md

```diff
@@ -36,10 +36,16 @@
 * Added `sympy>=1.5` to the list of dependencies.
   Removed the `sympy.functions.atan2` workaround now that SymPy has been fixed.
   [#280](https://github.com/XanaduAI/strawberryfields/pull/280)
-  
+
```

Comment:
My text editor automatically removes trailing whitespace, hence the changes above.